packages/asset/contracts/Asset.sol

```diff
@@ -40,7 +41,8 @@ contract Asset is
     ERC1155SupplyUpgradeable,
     ERC1155URIStorageUpgradeable,
     OperatorFiltererUpgradeable,
-    MultiRoyaltyDistributer
+    MultiRoyaltyDistributer,
+    ITokenInfo
```

Comment:
Why would you not call this `ITokenUtils` ?

Reply:
I was a little confused regarding Utils vs Info. Info seemed a lot more descriptive name.

Reply:
We are using Utils in the other file, it makes sense for the naming to be aligned to Utils